src/components/Alert/theme.js

```diff
@@ -0,0 +1,12 @@
+export default {
+    primary: {
+        backgroundColor: '#031633',
+        color: '#6EA8FE',
+        borderColor: '#084298'
+    },
+    secondary: {
+        backgroundColor: '#161719',
+        color: '#A7ACB1',
+        borderColor: '#41464B'
+    }
+}
```

Comment:
W motywie zazwyczaj trzymamy wartości tj. wartość koloru, wielkość odstępu itp. jeśli będziesz trzymał całe style to okaże się, że cały CSS jest w motywie, a nie o to chodzi ;)

PS. To taka uwaga na przyszłość - nie chodzi o zmienianie tego teraz :)